### DISCUSSION TITLE
Gradient color stop opacity

### DISCUSSION THREAD

**Discussion by robertdrakedennis** [tailwindlabs/tailwindcss#4019] [Ideas]
I've been working with gradients a lot recently, and I realized there is no way to have specific stops in gradients. I was thinking of the language being something akin to:

``from-opacity-{amount?}`` ``via-opacity-{amount?}`` ``to-opacity-{amount?}``

```
ex:

<div class="bg-gradient-to-r from-red-500 from-opacity-50 to-green-500"></div>
```

**Comment by simonswiss** [comment]
Hey!

I don't see this happening at least until `jit` mode is the _default_, because adding opacity permutations for `from`, `via` and `to` for each available colour, and replicate that for each breakpoint... would add yet another truckload 🚚 of additional utilities.

In a future where `jit` is _the way_ Tailwind works, that could become something to consider, if we can identify there is a sufficient need for such granularity to justify adding it to core 👍
